arduino/include/arduino.h

```diff
@@ -0,0 +1,80 @@
+#include <iostream>
+#include <vector>
+#include <fstream>
+#include <string>
+#include "serialib.h"
+#include <ios>
+#include <thread>
+#include <chrono>
+#include <mutex>
+#include <ctime>
+#include <cstdio>
+#include <stdexcept>
+#include <memory>
```

Comment:
in the h file there should only be the includes that are being used in the h file.
every include that is being used in the cpp file should be in the cpp